exercise_solutions.tex

```diff
@@ -1441,6 +1441,245 @@ \subsection*{Solution to \cref{ex:acconn}}
+
+Now is a good time to make a drawing to convince oneself that this is the
+right way to define the inclusions between consecutive spheres, and that this
+is the diagram intended in he exercise.
+
+Lets define the type $\Sn^{\infty}$ as the colimit of the diagram we just
+constructed. The constructors are:
+\begin{align*}
+  j_{\blank} &: \prd{n : \nat} \Sn^n \to \Sn^{\infty}\\
+  \glue_{\blank} &: \prd{n : \nat} \prd{x: \Sn^n} j_n(x) = j_{n+1}\circ i_n (x)
+\end{align*}
+And the induction:
+\[
+  \ind{\Sn^{\infty}} : \prd{C:\Sn^\infty \to \UU} \prd{J_{\blank} : \prd{n : \nat} \prd{x : \Sn^n} C(j_n(x))}
+  \Parens{\prd{n:\nat} \prd{x:\Sn^n} \dpath C {\glue_n(x)} {J_n(x)} {J_{n+1}\circ i_n (x)}}
+  \to \prd{e:\Sn^{\infty}} C(e)
+\]
+
+We also have the usual computational rules.
+Now, we can interpret the proof that we are going to give as follows.
+Each sphere is the ``equator'' of the next one. We say ``equator'' because it passes
+trough $\north$ and $\south$ so technically you have to rotate the bigger sphere for
+the inclusion to be exactly in the equator.
+The intuitive idea is that we can retract to a point a circle that is the equator of a sphere.
+Just as we can retract to a point two points that are the equator of a circle.
+This idea extends to all $n$-spheres. If we have all the spheres at the same time
+we can retract them all. But we have to be careful: the homotopies
+must be coherent for the total homotopy to be well defined. This can be done in
+classical homotopy theory using the fact that a CW complex is the colimit of
+the inclusions between its skeletons. In our case we constructed the colimit.
+So, if we define a homotopy for each sphere, and prove that this functions
+coincide in the glued parts (respect the equalities given by $\glue$), by
+induction on the colimit we get a homotopy defined in the colimit.
+
+The outline of the argument is as follows. It suffices to show that every point in
+$\Sn^{\infty}$ is equal to $j_0(\north_0)$, we prove it in two steps.
+The first step is to prove it only for the points of the form $j_n(\north_n)$.
+For the second step we will take an arbitrary $x:\Sn^{\infty}$
+and, by induction, we will assume it comes from a $y : \Sn^n$ for some $n$,
+that is $x \defeq j_n(y)$.
+Then we note that using the commutative diagram given by $\glue_n(x)$:
+\begin{center}
+\begin{tikzpicture}
+\node (N0) at (0,2) {$\Sn^{n}$};
+\node (N1) at (2,2) {$\Sn^{n+1}$};
+\node (N2) at (2,0) {$\Sn^{\infty}$};
+\node (N3) at (1.1,1.1) {};
+\node (N4) at (1.5,1.5) {};
+\draw[->] (N0) -- node[above]{\footnotesize $i_n$} (N1);
+\draw[->] (N0) -- node[left]{\footnotesize $j_n$} (N2);
+\draw[->] (N1) -- node[right]{\footnotesize $j_{n+1}$} (N2);
+\draw[double, double equal sign distance] (N3) -- node[left,above]{\footnotesize $glue_n$} (N4);
+\end{tikzpicture}
+\end{center}
+we get a path $x \defeq j_n(y) = j_{n+1} \circ i_n(y)$.
+But, as we will show, the image of $\Sn^n$ under $i_n$ is contractible,
```

Comment:
"the image of $\Sn^n$ under $i_n$ is contractible" is a little imprecise, which I suppose is okay since this is only an outline, but I think it might confuse the reader.  What you mean is that the "inclusion" `i_n` is nullhomotopic, right?